stories/example-screen-form-view.stories.js

```diff
@@ -0,0 +1,451 @@
+  <div class="card bg-light border-0 mb-2">
+    <div class="card-header bg-light mb-0 rounded">
+      <div class="d-flex justify-content-between">
+        <button class="d-flex text-left btn text-warning text-decoration-none p-0 mb-0" data-bs-toggle="collapse" href="#collapseInfoFour" aria-expanded="false" aria-controls="collapseInfoFour">
+          <span class="tabler-chevron-down lh-1 my-0 me-1">
+            <svg width="1rem" height="1rem">
+              <use xlink:href="${iconSprite}#tabler-chevron-down" />
+            </svg>
+          </span>
+          <span class="fs-5 fw-bolder lh-sm">
+            Meetings
+          </span>
+        </button>
+      </div>
+    </div>
+    <div class="collapse show" id="collapseInfoFour">
+      <div class="card-body text-left pt-0 fs-6">
+        <div class="d-flex justify-content-between mb-2">
+          <svg width="1.125rem" height="1.125rem" class="mt-1 me-2" role="button" tabindex="0">
+            <use xlink:href="${iconSprite}#tabler-file-search" />
+          </svg>
+          <div class="d-flex flex-column w-100">
+            <div>
+              <span class="me-1">Date:</span>
+              <span class="text-muted">2019-06-14 12:03:00</span>
+            </div>
+            <div>
+              <span class="me-1">Status:</span>
+              <span class="text-muted">Meeting Scheduled</span>
+            </div>
+          </div>
+          <div class="d-flex justify-content-right">
+            <span class="fs-6 fw-light" role="button" tabindex="0">Edit</span>
+          </div>
+        </div>
+        <div class="d-flex justify-content-between mb-2">
+          <svg width="1.125rem" height="1.125rem" class="mt-1 me-2" role="button" tabindex="0">
+            <use xlink:href="${iconSprite}#tabler-file-search" />
+          </svg>
+          <div class="d-flex flex-column w-100">
+            <div>
+              <span class="me-1">Date:</span>
+              <span class="text-muted">2019-06-14 12:03:00</span>
+            </div>
+            <div>
+              <span class="me-1">Status:</span>
+              <span class="text-muted">Meeting Scheduled</span>
+            </div>
+          </div>
+          <div class="d-flex justify-content-right">
+            <span class="fs-6 fw-light" role="button" tabindex="0">Edit</span>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+`
+
+const tabsNavigationHtml = `
+  <div class="w-100 overflow-auto p-0 mb-0 border border-light">
+    <div class="bg-white rounded-top border border-light d-flex">
+      <button class="d-flex align-items-center bg-white border-0 text-light rounded-top">
+        <svg class="icon-lg-font-size">
+          <use xlink:href="${iconSprite}#tabler-chevron-left" />
+          <span class="visually-hidden">Extend tabs' options</span>
+        </svg>
+      </button>
+      <ul class="w-100 overflow-hidden flex-nowrap nav nav-tabs nav-tabs-single-row-with-controls"></ul>
```

Comment:
There is a combination of things going on with the nav that is breaking the layout. Let's remove all occurrences of `w-100` as they are not doing anything currently (there's several in the file, remove all).

Then let's change this line, like so:
```
<ul class="overflow-hidden nav nav-tabs nav-tabs-single-row-with-controls" style="height: 60px;">
```

broken:
![2021-09-08_08-39-35AM](https://user-images.githubusercontent.com/1605754/132522163-bba6b6b7-2dad-41b0-a119-23f9b506fd8c.png)

fixed:
![2021-09-08_08-50-01AM](https://user-images.githubusercontent.com/1605754/132522179-321b719d-4208-4efc-9863-1c5a08bf1da4.png)